paymentsheet/src/test/java/com/stripe/android/paymentsheet/paymentdatacollection/cvcrecollection/DefaultCvcRecollectionInteractorTest.kt

```diff
@@ -0,0 +1,76 @@
+package com.stripe.android.paymentsheet.paymentdatacollection.cvcrecollection
+
+import app.cash.turbine.test
+import com.google.common.truth.Truth.assertThat
+import com.stripe.android.model.CardBrand
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.test.UnconfinedTestDispatcher
+import kotlinx.coroutines.test.resetMain
+import kotlinx.coroutines.test.runTest
+import kotlinx.coroutines.test.setMain
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+
+class DefaultCvcRecollectionInteractorTest {
+
+    @Before
+    fun setup() {
+        Dispatchers.setMain(UnconfinedTestDispatcher())
```

Comment:
something we do in some interactors which makes testing easier is making the interactor take a dispatcher as one of its params. Then, you can pass a test dispatcher in tests and control it, which is convenient. 

example: [DefaultManageScreenInteractorTest](https://github.com/stripe/stripe-android/blob/6e060b6239a82efd486ffd9e1292bcfff7c39a85/paymentsheet/src/test/java/com/stripe/android/paymentsheet/verticalmode/DefaultManageScreenInteractorTest.kt#L4)

Reply:
not sure if this is needed, but just as a tip!

Reply:
I'll take them out, looks like we don't actually need it